gui/projectfiledialog.cpp

```diff
@@ -329,6 +329,7 @@ void ProjectFileDialog::loadFromProjectFile(const ProjectFile *projectFile)
     mUI->mComboBoxPlatform->setCurrentText(projectFile->getPlatform());
     setSuppressions(projectFile->getSuppressions());
 
+    // TODO
```

Comment:
Why not remove the comment block?

Reply:
Because I didn't look into why it is commented.